topics/deta/index.md

```diff
@@ -0,0 +1,10 @@
+---
+created_by: Abstract Computing UG in Berlin
+display_name: Deta
+github_url: https://github.com/deta/
+logo: deta.png
+short_description: A Cloud for the Next Billion Ideas. Build your apps in hours, deploy them in seconds.
+topic: deta
+url: https://www.deta.sh/
+---
+Deta makes serverless development easy. They offer free serverless hosting and NoSQL databases. Check out their Python, JavaScript, and Go SDKs for optimal usage.
```

Comment:
Could this be reworded to sound a bit less like advertising? Thanks ❤️

Reply:
Sure, I'll try!

Reply:
Done!